libubx/rtlog.h

```diff
@@ -35,7 +35,7 @@ enum {
 
 #define ubx_block_log(level, b, fmt, ...)				\
 	if (b->loglevel && level <= *b->loglevel)					\
-		ubx_log(level, b->ni, b->name, fmt, ##__VA_ARGS__);	\
+		__ubx_log(b->ni, level, b->name, fmt, ##__VA_ARGS__);	\
```

Comment:
OK, so the above will log unconditionally, if the block loglevel permits it. But if no per block loglevel is defined, it should fall back on the node loglevel and not stay quiet:

```C
if (b->loglevel) {
    if (level <= *b->loglevel)
       __ubx_log(...)
} else {
    if (level <= ni->loglevel)
        __ubx_log(...)
}
```